docgen/src/guides/v3-migration.md

```diff
@@ -4,17 +4,98 @@ InstantSearch 3 introduces some breaking changes in the widget's naming, options
 
 ## InstantSearch
 
-### URLSync is not supported anymore
+### `appId` and `apiKey` are replaced by `searchClient`
+
+### Previous usage
+
+1.  [Import `InstantSearch.js`](https://community.algolia.com/instantsearch.js/v2/getting-started.html#install-instantsearchjs)
+2.  Initialize InstantSearch
+
+```javascript
+const search = instantsearch({
+  indexName: 'indexName',
+  appId: 'appId',
+  apiKey: 'apiKey',
+});
+
+search.start();
+```
+
+### New usage
+
+1.  [Import `algoliasearch`](https://github.com/algolia/algoliasearch-client-javascript)
```

Comment:
Prefer the documentation link rather than the GitHub page. We can also advice users to use the lite version of the client rather than the regular one. Otherwise they will have an increase of the size.

> https://www.algolia.com/doc/api-client/javascript/getting-started/